src/components/CartFilled.vue

```diff
@@ -23,6 +26,16 @@
 
 
 export default {
+  data () {
+    return {
+    showCart: false
+    }
+  },
+  methods : {
+    deleteCart () {
+
+    }
```

Comment:
[question] Why is there an empty method here?

Reply:
I have not finished working on the functionalities of the app, would be completing that.